test/parallel_api/ranges/sort_ranges_sycl.pass.cpp

```diff
@@ -53,6 +54,29 @@ main()
 
     bool res2 = ::std::is_sorted(data2, data2 + max_n, ::std::greater<int>());
     EXPECT_TRUE(res2, "wrong effect from 'sort with comparator' with sycl ranges");
+
+    //test with random number and projection usage
+    std::default_random_engine gen{std::random_device{}()};
```

Comment:
Some places are started with `::std` and others with `std`. I believe it should be `::std` but please let me know if you think otherwise.

Reply:
I suggest the change as a separate patch just for `::std`, otherwise we have got no-relevant massive changed lines in the current PR - it is not good IMHO.

Reply:
At least for test file they could be changed.

Reply:
I think for tests there is no requirement to use `::std`. But being consistent at least within a file is of course desirable.